document/core/appendix/properties.rst

```diff
@@ -62,24 +62,24 @@ Results
 :ref:`Results <syntax-result>` :math:`\val^\ast~(\THROWadm~\tagaddr)`
 .....................................................................
 
-* The :ref:`tag address <syntax-tagaddr>` :math:`\tagaddr` must be in :math:`\moduleinst.\MITAGS`.
-
 * The :ref:`external value <syntax-externval>` :math:`\EVTAG~\tagaddr` must be :ref:`valid <valid-externval-tag>` with some :ref:`external type <syntax-externtype>` :math:`\ETTAG~\tagtype`.
 
-* Let :math:`[t_1^\ast]\to[]` be the :ref:`tag type <syntax-tagtype>` |tagtype|.
+* Let :math:`[t^\ast]\to[{t'}^\ast]` be the :ref:`tag type <syntax-tagtype>` |tagtype|.
```

Comment:
There seems to be a step missing after this one that checks that `t'*` is empty.